TLM/CSModLib/GameObjects/NetInfoExtensions.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace CSModLib.GameObjects {
```

Comment:
this is a lot like `TrackUtil.cs`

Reply:
not sure but `IsRoadLane` might be ambiguous because does not tell us if it is only road lane or it includes road lane (like mixed tram+car lane). That is why I chose to use `MatchesRoad()` and `IsRoadOnly()`  to differentiate.